CHANGES.rst

```diff
@@ -14,6 +14,8 @@ https://github.com/zopefoundation/Zope/blob/4.0a6/CHANGES.rst
 New features
 ++++++++++++
 
+- add zconsole module for running scripts and interactive mode
```

Comment:
Nitpick: This is an Englisch sentence it could start with an upper letter and end with a ".".